### PR TITLE
Add notes on safe remote Dom0 terminals

### PR DESCRIPTION
**Pull request by jpouellet**

### CODE REVIEW

File: debugging/safe-remote-ttys.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+A note on serial consoles
+-------------------------
+
+If your machine has a serial console, you may with to use that, but note that a similar split-I/O model should be used to ensure Dom0 integrity.
+If you use the serial console as normal (via e.g. getty on ttyX, and logging in as normal), then the machine at the end of the serial cable could compromise your machine!
+Ideally, you would take input from your trusted keyboard, and only send the output over the serial cable via e.g. disabling getty and using:
+
+~~~
+script -f /dev/tty0
```
Comment (marmarek) [inline debugging/safe-remote-ttys.md]: `/dev/ttyS0`?

Comment (jpouellet) [inline debugging/safe-remote-ttys.md]: Err... yeah. I think you're right. Will change & link.

**Comment by jpouellet** [comment]
Done. Also added missing metadata.

**Comment by jpouellet** [comment]
Interesting how github still shows changes as approved even after I add more commits... :confused:

**Comment by marmarek** [comment]
> Interesting how github still shows changes as approved even after I add more commits... :confused:

Yes, this is how it works... But at least it is visible on the timeline what commits were approved.

**Pull request merged**